<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>account</class>
 <widget class="QDialog" name="account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="displayNameLabel">
     <property name="text">
      <string>Display Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="hostLabel">
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="xEnabled">
     <property name="text">
      <string>X Forwarding</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="host"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="remoteDirectoryLabel">
     <property name="text">
      <string>Remote Dir</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="remoteDirectory"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="ports">
     <property name="styleSheet">
      <string notr="true">QGroupBox { border: 1px solid black; }</string>
     </property>
     <property name="title">
      <string>Port Forwarding</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QCheckBox" name="localEnabled">
        <property name="text">
         <string>Local</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="socksEnabled">
        <property name="text">
         <string>SOCKS</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="remoteEnabled">
        <property name="text">
         <string>Remote</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="localPort"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="remotePort"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="socksPort"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="compressionEnabled">
     <property name="text">
      <string>Compression</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="displayName"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="username"/>
   </item>
   <item row="6" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>account</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>account</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
